week-1-practise/level-2/2-format-currency.js

```diff
@@ -6,7 +6,9 @@
   Write a function that transforms numbers into the format £0.00
 */
 
-function formatCurrency() {}
+function formatCurrency(price) {
+  return "£" + price.toFixed(2);
```

Comment:
Good stuff! What happens if someone used this function with a string? Try it out :)

Reply:
I'm not sure I understand your question.